js/handlers.js

```diff
@@ -29,6 +29,15 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
         new_date.setHours(0, 0, 0, 0);
         // Check if the date changed
         if (old_date.valueOf() !== new_date.valueOf()) {
+            var supports_formatting_date = false;
+            var should_be_tuesday = new Date(Date.UTC(2012,11,12,3,12,0))
+                .toLocaleDateString("en-US-u-ca-iso8601", {"weekday":"long"});
```

Comment:
as in the MDN's page function? I'd be pretty okay with that. I don't know 100% for sure that `not throwing a RangeError \implies .toLocaleDateString works correctly`, but I'm like 75% sure.

So just to confirm, the `supported()` function from MDN?